tensorflow/c/kernels.h

```diff
@@ -128,6 +133,11 @@ TF_CAPI_EXPORT extern void TF_DeleteKernelBuilder(TF_KernelBuilder* builder);
 // --------------------------------------------------------------------------
 // OpKernelContext routines
 
+// TF_GetStream returns the SP_Stream available in ctx
+// This function is only for pluggable device
+// it will return nullptr in all other cased.
```

Comment:
Change description:
"This function is only for pluggable device it will return nullptr in all other cased."
to
"This function returns a stream only for devices registered using the StreamExecutor C API (https://cs.opensource.google/tensorflow/tensorflow/+/master:tensorflow/c/experimental/stream_executor/stream_executor.h). It will return nullptr in all other cases.
Experimental: this function doesn't have compatibility guarantees and subject to change at any time."

Reply:
done